p2p/src/peers/mod.rs

```diff
@@ -374,6 +374,21 @@ impl Peers {
 
         true
     }
+
+    /// Remove address from the ice bucket
+    pub fn ice_peer_address_remove(&mut self, address: &SocketAddr) {
+        log::trace!("Removing peer address {} from the ice bucket", address);
+
+        self.ice_bucket.remove(address);
+    }
+
+    /// Remove a list of addresses from the ice bucket
```

Comment:
Mention that this does not take into account the ice_period, and always removes the peers

```suggestion
    /// Remove a list of addresses from the ice bucket, ignoring the ice_period
```